plugins/microsoft_teams/plugin.spec.yaml

```diff
@@ -707,6 +707,15 @@ actions:
         type: string
         required: true
         example: InsightConnect Channel
+      role:
+        title: Member Role
+        description: Role of the member to add
+        type: string
+        enum:
+          - Owner
+          - Member
+        required: false
```

Comment:
Is it really non-required field? If so, please add the empty enum for an input, and change the code for API call 😃 

If it's required. Kindly please add the `default` field value (probably as a "Member")